Godeps/_workspace/src/github.com/Clever/gearadmin/Makefile

```diff
@@ -0,0 +1,20 @@
+SHELL := /bin/bash
+PKG = github.com/Clever/gearadmin
+SUBPKGS =
```

Comment:
What does this line do? does it just empty the list? If so the usage on line 4 can be removed?